minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -600,6 +602,97 @@ public synchronized void start() throws IOException, InterruptedException {
     if (executor == null) {
       executor = Executors.newSingleThreadExecutor();
     }
+
+    verifyUp();
+
+  }
+
+  private void verifyUp() throws InterruptedException, IOException {
+
+    Objects.requireNonNull(getClusterControl().managerProcess, "Error starting Manager");
+    Objects.requireNonNull(getClusterControl().managerProcess.info().startInstant().get(),
+        "Error starting Manager");
+
+    Objects.requireNonNull(getClusterControl().gcProcess, "Error starting GC");
+    Objects.requireNonNull(getClusterControl().gcProcess.info().startInstant().get(),
+        "Error starting GC");
+
+    int tsExpectedCount = 0;
+    for (Process tsp : getClusterControl().tabletServerProcesses) {
+      tsExpectedCount++;
+      Objects.requireNonNull(tsp, "Error starting TabletServer");
+      Objects.requireNonNull(tsp.info().startInstant().get(), "Error starting Manager");
```

Comment:
Should this read
``` 
"Error starting tserver"
```
and maybe add "instance not started" ?

Reply:
Addressed in 35efd1c